packages/slice-machine/components/Simulator/components/IframeRenderer/index.tsx

```diff
@@ -131,7 +128,9 @@ const IframeRenderer: React.FunctionComponent<IframeRendererProps> = ({
             minHeight: `${iframeSize.height}px`,
             minWidth: `${iframeSize.width}px`,
             overflowY: "auto",
-            transform: `scale(${getScaling(iframeSize, viewportSize)})`,
+            ...(viewportSize
+              ? { transform: `scale(${getScaling(iframeSize, viewportSize)})` }
+              : { display: "none" }),
```

Comment:
Wouldn't display none make the Iframe takes no space ?
Which would make the screen jump a little bit ?

Reply:
hmmm 🤔
Ok, just wanted to make sure because unless the Flex says the elements should align at the end, I thought it would jump as there would only be the editor centered in the middle of the Div.